frontend/src/Editor/QueryManager/QueryEditors/Restapi/TabContent.jsx

```diff
@@ -92,6 +93,28 @@ export default ({
             />
           </div>
         )}
+        {!bodyToggle && (
```

Comment:
Instead of adding negative check can we add ternary check at line 81 and add the below code in the else condition?